plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -1508,6 +1513,48 @@ public void testUpdateAfterEqualityDelete()
         }
     }
 
+    @Test
```

Comment:
Could you add a test to `BaseIcebergConnectorSmokeTest`? We want to to verify the behavior on all catalogs.

Reply:
I encountered some problems: I did not successfully load the `Basetable` in `BaseIcebergConnectorSmokeTest` (because I need to modify the table properties, but I cannot modify properties through sql) or is there any other way?